boswatch/utils/paths.py

```diff
@@ -17,13 +17,21 @@
 import logging
 import os
 import sys
+import platform
 
 logging.debug("- %s loaded", __name__)
 
-# todo searching for root part is not a nice solution atm
+# note searching for root part is not a nice solution atm
 ROOT_PATH = os.path.dirname(sys.modules['boswatch'].__file__).replace("\\", "/") + "/../"
-LOG_PATH = ROOT_PATH + "log/"
-CONFIG_PATH = ROOT_PATH + "config/"
+
+# implements a system adaption for the paths
+if platform.system() == "Linux":
+    LOG_PATH = "/var/log/boswatch/"
```

Comment:
Der Pfad wird nicht genutzt aktuell.
Momentaner Stand ist, dass der Pfad für die Logfiles hart codiert in der config INI der logger steht

Reply:
insert fixme in code